src/lib.rs

```diff
@@ -130,21 +130,20 @@ impl AllChargerLocations {
                     }
                 }
             }
-            if i % 1_000 == 0 {
-                println!("{:?} {}: {:?}", thread, i, start.elapsed());
-                println!("{:?} reachable: {}", thread, reachable);
-                println!("{:?} unreachable: {}", thread, unreachable);
-                println!("{:?} maybe reachable: {}", thread, maybe_reachable);
-                println!("{:?} api call count: {}", thread, api_call_counter);
-                api_call_counter = 0;
-            }
+            // if i % 1_000 == 0 {
+            //     println!("{:?} {}: {:?}", thread, i, start.elapsed());
+            //     println!("{:?} reachable: {}", thread, reachable);
+            //     println!("{:?} unreachable: {}", thread, unreachable);
+            //     println!("{:?} maybe reachable: {}", thread, maybe_reachable);
+            //     println!("{:?} api call count: {}", thread, api_call_counter);
+            //     api_call_counter = 0;
+            // }
```

Comment:
i commented this out rather than removing it, because i thought it seemed like we *might* eventually need to put it back for debugging purposes.